contracts/system/abi.go

```diff
@@ -357,3 +388,7 @@ func ABIPack(contract common.Address, method string, args ...interface{}) ([]byt
 func GetStakingABI(blockNum *big.Int, config *params.ChainConfig) abi.ABI {
 	return ABI(StakingContract)
 }
+
+func GetDAOCharityFoundationABI(blockNum *big.Int, config *params.ChainConfig) abi.ABI {
```

Comment:
This function may not be needed, because it seems not to have potential forks,
Use `ABI()` or `ABIPack()` directly instead